src/Mod/Sketcher/Gui/SketcherToolDefaultWidget.cpp

```diff
@@ -94,27 +94,32 @@ bool SketcherToolDefaultWidget::eventFilter(QObject* object, QEvent* event)
 void SketcherToolDefaultWidget::parameterOne_valueChanged(double val)
 {
     Q_UNUSED(val);
-    QMetaObject::invokeMethod(ui->parameterTwo, "setFocus", Qt::QueuedConnection);
+    isSet[0] = 1;
+    setParameterFocus(1);

```

Comment:
Much better :) It is immediately known what is intended. 

Now, it is generally undesirable to have hard-coded numbers, which are generally called "magic numbers". Two options are generally used for this, to use constants or enums.

Do not make any change. This is just for information at this point.